doc/getting-started.rst

```diff
@@ -1,90 +1,100 @@
-
-.. note:: ``myfirstapp`` stands for the directory (or path) where the project
-          will reside in, if omitted, ``cabal init`` will do its proceedings
-          in the directory it's called in.
-
-.. note:: ``-n`` stands for ``--non-interactive``, which means that cabal will try to guess
-          how to set up the project for you and use the default settings, which will serve us
-          well for the purpose of this tutorial.
-          When setting up your projects in the future, you will likely want to omit ``-n``
-          and do just ``cabal init``, so that cabal will interactively ask you
-          for the details on how the project should be set up
-          (while still offering reasonable defaults on every step).
-          Also, you can run ``cabal init --help`` to get more info on how ``cabal init`` can be used.
+    $ cabal init myapp --non-interactive
 
-This will generate the following files:
+in a terminal. This generates the following files in a new ``myapp`` folder:
 
 .. code-block:: console
 
     $ tree
     .
-    └── myfirstapp
+    └── myapp
         ├── app
         │   └── Main.hs
         ├── CHANGELOG.md
-        └── myfirstapp.cabal
+        └── myapp.cabal
 
-``app/Main.hs`` is where your package's code lives.
+``myapp.cabal`` is Cabal's metadata file which describes your package,
+its dependencies and how it is built:
 
-``myfirstapp.cabal`` is Cabal's metadata file which describes your package,
-how it is built and its dependencies. We'll be updating this file in a
-little bit when we add an external dependency to our package.
+.. code-block:: cabal
 
+    cabal-version:      3.0
+    name:               myapp
+    version:            0.1.0.0
+    -- ...
 
-Running the application
-^^^^^^^^^^^^^^^^^^^^^^^
+    executable myapp
+        import:           warnings
+        main-is:          Main.hs
+        build-depends:    base ^>=4.19.0.0
+        hs-source-dirs:   app
+        default-language: Haskell2010
 
-When we ran ``cabal init myfirstapp -n`` above, it generated a package with a single
-executable named same as the package (in this case ``myfirstapp``) that prints
-``"Hello, Haskell!"`` to the terminal. To run the executable enter the project's
-directory and run it, by inputting the following commands:
+The ``executable`` section shows that this package's build output 
+is an executable called ``myapp`` that we can run on the command line. 
+The ``build-depends`` section contains a list of all package dependencies.
 
-.. code-block:: console
 
-    cd myfirstapp
-    cabal run myfirstapp
+``app/Main.hs`` is where your executable's code lives:
+
+.. code-block:: haskell
+
+    module Main where
+
+    main :: IO ()
+    main = putStrLn "Hello, Haskell!"
+
 
-You should see the following output in the terminal:
+To run the executable, switch into the project folder with ``cd myapp`` and  run 
 
 .. code-block:: console
 
-     $ cabal run myfirstapp
+     $ cabal run
```

Comment:
I'm not sure I'm a big fan of this change. I'd rather (1) put the component's name back and (2) mention in the text that with one executable in the package, `cabal run` is smart enough to infer it, so the name can be omitted.

Reply:
Done